sites/page-details/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,593 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+  "@type": [
+    "earl:Assertion",
+    "acr:AssessmentReport"
+  ],
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://example.com/web/page/url.html",
```

Comment:
I think this page was pre-evaluated:https://wet-boew.github.io/GCWeb/sites/page-details/page-details-pft-en.html